mint.json

```diff
@@ -111,7 +111,8 @@
         "cerebrium/endpoints/websockets",
         "cerebrium/endpoints/webhook",
         "cerebrium/endpoints/async",
-        "cerebrium/endpoints/custom-web-servers"
+        "cerebrium/endpoints/custom-web-servers",
+        "cerebrium/endpoints/custom-dockerfiles"
```

Comment:
I dont think this is the right place to put it but rather under Containers and making the title:
- Dockerfile
- Bring your own Dockerfile
- etc